app/renderers/hyrax/renderers/external_url_attribute_renderer.rb

```diff
@@ -1,20 +1,25 @@
 class ExternalUrlAttributeRenderer < Hyrax::Renderers::AttributeRenderer
+
   def render
     markup = ''
     values.delete("") if values # delete an empty string in array or it would display
     return markup if values.blank? && !options[:include_empty]
-    link = values.is_a?(Array) ? values.join : values
-    li_value(link)
+    return arr_of_li_values(values) if values.is_a?(Array)
+    Rails.logger.info("ExternalUrlAttributeRenderer did not render #{values}")
```

Comment:
Remove the Rails logger on line 7 added  for debugging. 
Also since in Ruby  a method will return value of the last evaluated statement, it means what is being returned is the Rails logger output.